x-pack/plugins/secops/public/components/page/hosts/host_summary/index.test.tsx

```diff
@@ -102,19 +110,23 @@ describe('Host Summary Component', () => {
       );
       const wrapper = mountWithIntl(
         <ReduxStoreProvider store={store}>
-          <DragDropContextWrapper>{euiDescriptionList}</DragDropContextWrapper>
+          <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
+            <DragDropContextWrapper>{euiDescriptionList}</DragDropContextWrapper>
+          </ThemeProvider>
         </ReduxStoreProvider>
       );
       expect(wrapper.text()).toBe(
-        'Namesiem-kibanaLast BeatyesterdayIdaa7ca589f1b8220002f2fc61c64cfbf1IP Address10.142.0.7fe80::4001:aff:fe8e:7MAC Addr42:01:0a:8e:00:07Typeprojects/189716325846/machineTypes/n1-standard-1PlatformdebianOS NameDebian GNU/LinuxFamilydebianVersion9 (stretch)Architecturex86_64'
+        'Namesiem-kibanaLast Beat--Idaa7ca589f1b8220002f2fc61c64cfbf1IP Address10.142.0.7fe80::4001:aff:fe8e:7MAC Addr42:01:0a:8e:00:07Typeprojects/189716325846/machineTypes/n1-standard-1PlatformdebianOS NameDebian GNU/LinuxFamilydebianVersion9 (stretch)Architecturex86_64'
```

Comment:
As above, in effort to simplify these tests, I went ahead and removed the `firstSeen` value from the mock data and am now just checking for the default value in its absence since this test is for verifying that an `EuiDescriptionList` renders properly.